src/main/java/uk/gov/hmcts/reform/slc/services/FtpAvailabilityChecker.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.slc.services;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalTime;
+
+@Component
+public class FtpAvailabilityChecker {
+
+    private final LocalTime downtimeStart;
+    private final LocalTime downtimeEnd;
+
+    public FtpAvailabilityChecker(
+        @Value("${ftp.downtime.from}") String downtimeFromHour,
+        @Value("${ftp.downtime.to}") String downtimeToHour
+    ) {
+        this.downtimeStart = LocalTime.parse(downtimeFromHour);
+        this.downtimeEnd = LocalTime.parse(downtimeToHour);
+    }
+
+    public boolean ftpAvailable(LocalTime time) {
+        if (downtimeStart.isBefore(downtimeEnd)) {
```

Comment:
It might be worth including the case when downtimeStart is equal to downtimeEnd, which would mean there's no downtime at all.